lib/core-common/src/utils/load-custom-babel-config.ts

```diff
@@ -48,10 +49,16 @@ function loadFromPath(babelConfigPath: string): TransformOptions {
       throw error.js;
     }
 
-    config = { ...config, babelrc: false };
+    if (config instanceof Function) {
+      config = config();
```

Comment:
Calling the config function without an argument could be problematic. Babel calls the function with an `api` object https://babeljs.io/docs/en/config-files#config-function-api
In my opinion it is ok to stub that api and make sure that calls to the methods don't fail. The `env` method should be easy to reimplement.